.github/workflows/stale.yml

```diff
@@ -18,11 +14,32 @@ jobs:
     if: github.repository == 'github/docs-internal' || github.repository == 'github/docs'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/stale@28ca1036281a5e5922ead5184a1bbf96e5fc984e # v9.0.0
+      - name: Mark stale issues and PRs (docs-internal)
+        if: github.repository == 'github/docs-internal'
+        uses: actions/stale@28ca1036281a5e5922ead5184a1bbf96e5fc984e # v9.0.0
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: 'This issue is stale because there have been no updates in 365 days.'
+          stale-pr-message: 'This PR has been automatically marked as stale due to 14 days of inactivity. Please update soon to avoid closure.'
```

Comment:
```suggestion
          stale-pr-message: 'This PR has been automatically marked as stalled after 14 days of inactivity (open PRs rapidly collect merge conflicts). Please update soon to avoid closure.'
```